### PR TITLE
Documentation for Self Hosted release v1.1.0

### PR DESCRIPTION
**Pull request by TheMacies**
# Description of the change

## Checklist

Please make sure that the proposed change checks all the boxes below before requesting a review:

- [ ] I have reviewed the [guidelines for contributing](https://github.com/spacelift-io/user-documentation/blob/main/CONTRIBUTING.md) to this repository.
- [ ] The preview looks fine.
- [ ] The tests pass.
- [ ] The commit history is clean and meaningful.
- [ ] The pull request is opened against the `main` branch.
- [ ] The pull request is no longer marked as a draft.
- [ ] You agree to license your contribution under the [MIT license](https://github.com/spacelift-io/user-documentation/blob/main/LICENSE) to Spacelift (not required for Spacelift employees).
- You have updated the navigation files correctly:
    - [ ] No new pages have been added, or;
    - [ ] Only _nav.yaml_ has been updated because the changes only apply to SaaS, or;
    - [ ] Only _nav.self-hosted.yaml_ has been updated because the changes only apply to Self-Hosted, or;
    - [ ] Both _nav.yaml_ and _nav.self-hosted.yaml_ have been updated.

If the proposed change is ready to be merged, please request a review from `@spacelift-io/solutions-engineering`. Someone will review and merge the pull request.

_Spacelift employees should request reviews from the relevant engineers and are allowed to merge pull requests after they got at least one approval._

Thank you for your contribution! 🙇

### CODE REVIEW

File: docs/product/administration/changelog.md [hunk 1]
```diff
@@ -4,6 +4,19 @@ description: Find out about the latest changes to the Self-Hosted Spacelift.
 
 # Changelog
 
+## Changes between v1.0.0 and v1.1.0
+
+### Features
```
Comment (adamconnelly) [inline docs/product/administration/changelog.md:9]: What about stack creation v2? That's mentioned in the release spreadsheet, or does it not seem like a big enough change to mention?

**Review by adamconnelly — DISMISSED** [review]
Just approving in case you decide not to include it.

File: docs/product/administration/changelog.md (hunk 1 above)
Comment (TheMacies) [reply docs/product/administration/changelog.md:9]: I think the doc is wrong. AFAIK it's been deployed to SH in 1.0.0 🤔

Comment (TheMacies) [reply docs/product/administration/changelog.md:9]: No, you are right.

Comment (TheMacies) [reply docs/product/administration/changelog.md:9]: Added

**Review by adamconnelly — APPROVED** [review]

**Pull request merged**